terraform/modules/vpc-hub/main.tf

```diff
@@ -288,6 +288,27 @@ resource "aws_route" "public-internet-gateway" {
   route_table_id         = aws_route_table.public.id
 }
 
+resource "aws_route" "public_mp_10-20-0-0--16" {
```

Comment:
Could we change the names of these to reflect what they are please? eg:
public_mp_core
public_mp_dev_test
public_mp_preprod_prod